### ISSUE TITLE
Cannot open terminal

### ISSUE THREAD

**Issue by Xymanek** [the-machine-project/the-machine#34]
The title says it all. It is impossible to open the terminal. Tried clicking around, smashing keyboard and no results. Running last code version downloaded from github (that is e971c2b), compiled manually

**Comment by devloop0** [comment]
My bad, the keyboard shortcut should be Control + Alt + T, like in Ubuntu. I should probably write this down somewhere. Let me know if this works and I'll close this issue.

**Comment by Xymanek** [comment]
Yep it works. I suggest you to leave this issue open till you write it down somewhere ;)

BTW the font is horrible (Windows 10 x64 here)

**Comment by devloop0** [comment]
I'll leave it open for now. Also, the font is a known issue, the only alternative is to switch to the Samaritan one for the Machine terminal as well. I thought it fit the theme better as well. This isn't really an issue on our end, unless we can find a better, similar-looking font. Otherwise, we might have to stick with the one we already have.

**Comment by RoccoDeveloping** [comment]
I cannot open terminal even if I do Ctrl + Alt + T. Win 10 x64 (Laptop). Where must I do that shortcut? Basically, the Machine loads up with face recognition. Then I press CTRL + Alt + T but nothing.

**Comment by RoccoDeveloping** [comment]
Wait. Just tried to decompile your jar and it seems that in your x64 download, there's not the showTerminal() check. Trying to compile & build myself and see if works

**Comment by devloop0** [comment]
If you are downloading from the website, we have not uploaded the latest builds there. We will soon. The only way to currently test the latest machine is to compile manually.

**Comment by gateslm** [comment]
Which commands currently work for the terminal?

**Comment by devloop0** [comment]
Type in 'help' in the terminal. It lists all the commands and their usages.

**Comment by RoccoDeveloping** [comment]
Yeah! It works, but now... look at my new issue!
